config.json

```diff
@@ -178,6 +178,13 @@
       "topics": [
 
       ]
+    },
+    {
+      "difficulty": 2,
+      "slug": "beer-song",
```

Comment:
I think this problem should fall much earlier in the track. The main aspect of this is string substitution/manipulation (which is not particularly difficult) and this is paired with looping & conditionals which should have been established pretty early on. Most of the other tracks (with a few exceptions) seem to have placed this problem somewhere in the first half of the track. We've got less exercises implemented though so for now I'd suggest somewhere around the middle near `anagram`/`pangram`/`isogram`.

I'm aware that the ordering of exercises for the track as a whole still needs more attention, but again given the upcoming redesign it's probably not worth agonizing over now - I did some re-ordering in [42](https://github.com/exercism/r/pull/42) based on a comparison of other language tracks and I noticed there was a lot of variability - so for now I think let's just aim for something 'reasonably' ordered.